ui/src/kapacitor/components/RuleHeader.js

```diff
@@ -41,30 +44,29 @@ export const RuleHeader = React.createClass({
     this.setState({isEditingName: !this.state.isEditingName});
   },
 
-
   render() {
     return (
       <div className="page-header">
         <div className="page-header__container">
           <div className="page-header__left">
             {this.renderEditName()}
           </div>
-          <div className="page-header__right">
-            {this.renderSave()}
-          </div>
+          {this.renderSave()}
```

Comment:
I like the look of this. but now it doesn't line up with `this.renderEditName`. Would you move the surrounding markup into that file too? Or if you have OCD medication for me that would also be acceptable